samples/containers/android/src/main/java/com/squareup/sample/container/overviewdetail/OverviewDetailContainer.kt

```diff
@@ -97,9 +99,9 @@ class OverviewDetailContainer(view: View) : LayoutRunner<OverviewDetailScreen> {
     stub.update(combined, viewEnvironment + (OverviewDetailConfig to Single))
   }
 
-  companion object : ViewFactory<OverviewDetailScreen> by LayoutRunner.Binding(
+  companion object : ViewFactory<OverviewDetailScreen> by LayoutRunnerViewFactory(
       type = OverviewDetailScreen::class,
-      layoutId = R.layout.overview_detail,
+      layoutId = layout.overview_detail,
```

Comment:
Any reason for this change in the import/prefix delineation? Seems odd.

Reply:
Accidental. Frickin' JetBrains.